### ISSUE TITLE
Unable to create or import provisioned concurrency config for a lambda alias

### ISSUE THREAD

**Issue by armandobelardo** [pulumi/pulumi-aws#1886]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

### Steps to reproduce

1. Create a Lambda function
2. Create a Lambda alias
3. Create a ProvisionedConcurrencyConfig
4. `pulumi up`

**Expected:** 

ProvisionedConcurrencyConfig is created

**Actual:**
```
    error: Preview failed: refreshing urn:pulumi:dev::cognito-pool::aws:lambda/provisionedConcurrencyConfig:ProvisionedConcurrencyConfig::pre-token-lambda-ProvisionedConcurrency: 1 error occurred:
        * error getting Lambda Provisioned Concurrency Config (arn:aws:lambda:us-east-2:$ACCOUNT_ID:function:$LAMBDA_NAME:$ALIAS_NAME): ValidationException:
        status code: 400, request id: $REQUEST_ID
```

Similarly, creating the ProvisionedConcurrencyConfig in AWS Console and attempting to import it gets the same exact error.

**Code:**
```
        lambda_alias = aws.lambda_.Alias(f"{name}-Alias",
            function_name=lambda_function.arn,
            function_version=lambda_function.version,
            opts=ResourceOptions(
                provider=provider,
            )
        )

        aws.lambda_.ProvisionedConcurrencyConfig(f"{name}-ProvisionedConcurrency",
            function_name=lambda_function.arn,
            provisioned_concurrent_executions=provisioned_concurrency,
            qualifier=lambda_alias.name,
            opts=ResourceOptions(
                provider=provider,
            )
        )
```

**Comment by guineveresaenger** [comment]
Hi @armandobelardo - thank you for filing and for finding the upstream issue. We'll probably be beholden to upstream fixes for this for now.

**Comment by mikhailshilkov** [comment]
The issue has been closed upstream. The fix should be available in Pulumi AWS 6.0+.
